types/node/http.d.ts

```diff
@@ -205,7 +205,9 @@ declare module "http" {
 
     // https://github.com/nodejs/node/blob/master/lib/_http_client.js#L77
     class ClientRequest extends OutgoingMessage {
-        aborted: number;
+        aborted: boolean;
+        host: string;
+        protocol: string;
```

Comment:
I think `host` and `protocol` (maybe also others)  shoudl be readonly. Technically you can write to `host` but it is not used by http so it makes so sense.

Reply:
Looks like a lot of fields can be made readonly. I will try it later in separate PR.